src/components/Table/TableHead/FilterHeaderRow/FilterHeaderRow.test.jsx

```diff
@@ -40,22 +40,6 @@ describe('FilterHeaderRow', () => {
     expect(columns).toHaveLength(1);
   });
 
-  it('input blur fires apply filter callback', () => {
-    const wrapper = mount(
-      <FilterHeaderRow
-        {...commonFilterProps}
-        ordering={[{ columnId: 'col1' }]}
-        columns={[{ id: 'col1' }]}
-      />
-    );
-    const desiredState = { col1: 'option1' };
-
-    wrapper.setState(desiredState);
-    wrapper.find('input').simulate('blur');
-
-    expect(commonFilterProps.onApplyFilter).toHaveBeenCalledWith(desiredState);
-  });
-
   it('text input clear button clears filter', () => {
```

Comment:
perhaps replace the deleted blur-test with a test that checks that apply fires on change?

Reply:
there is a testcase already for checking onChange, but I can add one for onKeyPress

Reply:
ugh, actually I can't simulate keyPress: https://github.com/enzymejs/enzyme/issues/441

Reply:
Yeah, anything keyboard related needs to ideally be tested in browser. I've opened issue https://github.com/IBM/carbon-addons-iot-react/issues/1203 for this